src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
```

Comment:
can we move this style to the `stars__star` class? why do we need to add it separately?

Reply:
we can't add it to stars_star class, but i added it to stars class